src/ert/run_models/run_arguments.py

```diff
@@ -0,0 +1,69 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@dataclass
+class SingleTestRunArguments:
+    simulation_mode: str = field(init=False, default="Single test run")
+
+    active_realizations: List[bool]
+    current_case: str
+    target_case: str
+    iter_num: Optional[int] = 0
```

Comment:
```suggestion
    override_iteration_number: Optional[int] = None
```

This is used for explicitly setting the iteration number, regardless of what it should actually be. Really bad behaviour imo but we can be more explicit on what it does.